packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/versions.doc.ts

```diff
@@ -64,6 +66,14 @@ Refer to the [migration guide](/docs/api/pos-ui-extensions/migrating) for more i
 - Removed in POS version: N/A
 - Release day: 1/6/2025
 
+### Important fix in POS 9.29.0
+
+- Fixed an issue where the \`FormattedTextField\` were rendering black input text in dark mode. It will now render the correct light color variant.
+
+### Important fix in POS 9.27.0
+
+- Fixed an issue where the \`Stack\` component was not rendering correctly. By default, \`Stack\` will now always wrap. Additionally, stacks using \`'block'\` will now compress horizontally. Use \`alignContents: 'stretch'\` to take all of the horizontal space. This will not have an impact on layout in previous POS versions.
```

Comment:
When we change things like this, we should be providing screenshots of the diff. Trying to parse words is incredibly hard. Seeing the diff in images is very easy.